### PR TITLE
QE-496: Upload agent scenario

### PR DESCRIPTION
**Pull request by allaco-dn**
- Upload agent example scenario added
- expect_error if not present by default is False

### CODE REVIEW

File: src/python/test/behave_tests/features/dx.feature [hunk 1]
```diff
@@ -20,7 +20,6 @@ Feature: 1: dx commands example
   Scenario: 1030_CLI file upload test, project select on login
     Given CLI login as user qa_admin_org1
     When CLI create new project
-    # TODO: remove expect error (by default no)
```
Comment (avigriyanova) [inline src/python/test/behave_tests/features/dx.feature:23]: expect_error is still there, is it according to plan? :D

File: src/python/test/behave_tests/features/upload_agent.feature [hunk 2]
```diff
@@ -0,0 +1,11 @@
+@upload_agent
+Feature: 2: Upload agent example
+
+  Scenario: 2000_upload file with upload agent
+    Given CLI login as user qa_admin_org1
+    And CLI create new project
+      | name      | --select |
+      | [default] |          |
+    And generate file big 2 Bytes named zzz.file
```
Comment (avigriyanova) [inline src/python/test/behave_tests/features/upload_agent.feature:9]: why not `generate big 2 Bytes file named zzz.file`?

File: src/python/test/behave_tests/features/dx.feature (hunk 1 above)
Comment (allaco-dn) [reply src/python/test/behave_tests/features/dx.feature:23]: Yes, to show that both are supported :D But it is not mandatory

File: src/python/test/behave_tests/features/upload_agent.feature (hunk 2 above)
Comment (allaco-dn) [reply src/python/test/behave_tests/features/upload_agent.feature:9]: It is an existing step, phrased like that...

Comment (avigriyanova) [reply src/python/test/behave_tests/features/upload_agent.feature:9]: weird by ok :D I guess it's not worth to fix it now

**Pull request merged**